ansible/roles/providers/tasks/aws.yml

```diff
@@ -33,6 +33,41 @@
   when:
     - ansible_distribution == "Flatcar"
 
+
+- name: install cloud-init deps
+  yum:
+    name: "{{ packages }}"
+    state: present
+    enablerepo: "{{ 'offline' if offline_mode_enabled else '' }}"
+    disablerepo: "{{ '*' if offline_mode_enabled else '' }}"
+  vars:
+    packages:
+      - cloud-utils-growpart
+      - "python{{ '3' if ansible_distribution_major_version|int >= 9 else '2'}}-pip"
+  when: ansible_os_family == "RedHat"
+
+- name: install cloud-init for RHEL
```

Comment:
From the test logs I see that this task isn't doing anything
https://github.com/mesosphere/konvoy-image-builder/actions/runs/12035969214/job/33556280321?pr=1233#step:9:3372
https://github.com/mesosphere/konvoy-image-builder/actions/runs/12035969214/job/33556275752?pr=1233#step:9:3274

And prior to this we aren't installing cloud-init anywhere, is this task needed at all?